ansible_collections/arista/avd/roles/eos_designs/templates/ip_addressing/mlag-ibgp-peering-ip-secondary.j2

```diff
@@ -0,0 +1 @@
+{% set mlag_ibgp_peering_ip.peer_ip = vrf.mlag_ibgp_peering_ipv4_pool | ansible.netcommon.ipaddr('subnet') | ansible.netcommon.ipmath((switch.mlag_switch_ids.secondary - 1) * 2) %}
```

Comment:
That's not the same formula as the one in the [current secondary templates](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/templates/ip_addressing/mlag-ip-secondary.j2) and it is leading to having a wrong IP set for the remote neighbor

```suggestion
{% set mlag_ibgp_peering_ip.peer_ip = vrf.mlag_ibgp_peering_ipv4_pool | ansible.netcommon.ipaddr('subnet') | ansible.netcommon.ipmath(((switch.mlag_switch_ids.primary - 1) * 2) + 1) %}
```